cloudify_docker_image/README.md

```diff
@@ -0,0 +1,15 @@
+# General Usage
+ - _Note: tested on a Ubuntu 16.04.3 LTS (Xenial Xerus) host_
+ 
+## Perquisites
+`sudo apt-get install -y docker`
+
+### Follow through these instructions
+1. `sudo usermod -aG docker ubuntu`
+1. logout & log back in
+1. check that `docker ps` command works
+
+## Usage
+`sh make_cfy_image.sh` - assuming you have an rpm file that is mentioned in the [Dockerfile](Dockerfile). If you don't, remove the `COPY ...` statement and the `RUN ...` statement that comes right after. Build and copy a rpm to the container ([like here](https://github.com/cloudify-cosmo/cloudify-manager-install/blob/CFY-7602-add-tests/packaging/docker/build_rpm.sh)). 
```

Comment:
Not sure I understand this bit. What does this mean?

Reply:
Old instructions, I have updated it.